src/addon/info.cpp

```diff
@@ -73,7 +90,9 @@ void addon_info::read(const config& cfg)
 	const config::const_child_itors& locales_as_configs = cfg.child_range("translation");
 
 	for(const config& locale : locales_as_configs) {
-		this->locales.push_back(locale["language"].str());
+		if(locale["supported"].to_bool(true))
+			this->locales.push_back(locale["language"].str());
+		this->info_translations.push_back(addon_info_translation(locale));
```

Comment:
In general, `emplace_back` is preferred over `push_back` for containers that support it to avoid the extra copy, but this is not strictly an issue on my book.

Reply:
Ok.

Reply:
Ok, will use it.